pages/windows/choco-uninstall.md

```diff
@@ -0,0 +1,27 @@
+# choco uninstall
+
+> Uninstall one or more packages with Chocolatey.
+
+- Uninstall one or more space-separated packages:
+
+`choco uninstall {{package(s)}}`
+
+- Uninstall a specific version of a package:
+
+`choco uninstall {{package}} --version {{version}}`
+
+- Confirm all prompts automatically:
+
+`choco uninstall {{package}} --yes`
+
+- Uninstall all versions of a package:
```

Comment:
Just curious - what is the default behavior ? Remove only latest version ?

Reply:
Weird, according to the docs the default is to remove the latest.

When testing just now, it appears to remove all versions by default. 🤔 So I guess maybe this example is redundant.

Reply:
Sure.